docs/semgrep-appsec-platform/tags.md

```diff
@@ -28,7 +28,9 @@ Add tags for specific projects in the Semgrep AppSec Platform through the follow
 To manage tags through Semgrep AppSec Platform, follow these steps:
 
 1. Go to the Semgrep AppSec Platform [Projects](https://semgrep.dev/orgs/-/projects) page.
-2. Click on the project's <i class="fa-solid fa-gear"></i> **gear** icon to enter project configuration page, and then add or remove tags.
+2. Find the project to which you want to add or remove tags, then click its **<i class="far fa-window-restore"></i> icon** under **Details**.
```

Comment:
```suggestion
2. Find the project you want to modify, then click its **<i class="far fa-window-restore"></i> icon** under **Details**.
```

Reply:
this is one of those places where both options for the prepositional phrase sound a bit weird (yours is very correct, but sounds formal, the other way leaves a hanging "to" that sounds strange), so I went for another phrasing entirely